building/gits/optee_os.rst

```diff
@@ -90,6 +90,29 @@ The same for an QEMU Armv8-A (AArch64) would look like this:
 
         $ make V=1
 
+Build using LLVM/clang
+======================
+Most of the code in :ref:`optee_os`.git can be compiled using llvm/clang.
+However, llvm-objdump does not support mixed 32-bit ARM and Thumb instructions,
+hence there is still a need for having gcc available for some of the build
+steps. Start build getting the toolchain (see :ref:`llvm`). After that you can
```

Comment:
Not a strict need, inf fact not having a GCC toolchain in the path should just cause warnings (and not create the `.dmp` files which are only debugging aids).

But I have better news yet: https://github.com/OP-TEE/optee_os/pull/3625

Reply:
Ok, then I simply remove this section.

Reply:
Then you can add my:

`Reviewed-by: Jerome Forissier <jerome@forissier.org>`